easybuild/easyblocks/b/basix.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+from easybuild.easyblocks.generic.cmakepythonpackage import CMakePythonPackage
+from easybuild.tools.run import run_cmd
+from easybuild.tools.build_log import EasyBuildError
+
+
+class EB_Basix(CMakePythonPackage):
+    """Custom easyblock for Basix"""
+    def build_step(self):
+        """Build step need not be overridden"""
+        super(EB_Basix, self).build_step()
+
+        # Building 'lib64/libbasix.so'
+        self.cfg['build_type'] = 'Release'
+        self.cfg['srcdir'] = 'cpp'
+        self.cfg['build_cmd'] = 'cmake --build %s/easybuild_obj' % self.builddir
```

Comment:
Why is this used? Especially "Build step need not be overridden" sounds like you don't need this whole method, do you?